uber/models/attendee.py

```diff
@@ -589,16 +589,17 @@ def calculate_badge_cost(self, use_promo_code=True):
         registered = self.registered_local if self.registered else None
         base_badge_price = self.base_badge_price or c.get_attendee_price(registered)
 
+        age_discountable_badge_types = [c.ATTENDEE_BADGE]
```

Comment:
I think this needs to be moved up to the class level, instead of being defined in the function.